pkg/parser/parser.go

```diff
@@ -33,6 +33,12 @@ import (
 	"github.com/crossplane/crossplane-runtime/pkg/errors"
 )
 
+// Lintable defines the common API for lintable packages.
+type Lintable interface {
+	GetMeta() []runtime.Object
```

Comment:
what is the semantic difference between these two methods, since they return the same types? Can you please add some doc?

Reply:
Added some comments, hope it makes it more clear!